sites/main-site/src/content/events/2025/hackathon-march-2025/crg.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 'Hackathon - March 2025 (CRG)'
+subtitle: 'Local node of the nf-core hackathon at CRG'
+shortTitle: 'CRG'
+type: 'hackathon'
+startDate: '2025-03-17'
+startTime: '09:00+01:00'
+endDate: '2025-03-19'
+endTime: '17:00+01:00'
+locations:
+  - name: CRG
+    address: |
+      PRBB building, C/Doctor Aiguader, 88
+    links:
+      - https://www.crg.eu/
+    geoCoordinates: [30.336332499999997, -81.51521803329344]
```

Comment:
![image](https://github.com/user-attachments/assets/af53ba30-fd2e-4a9c-b447-9a73e14bd6c9)

Maybe not :sweat_smile: . This probs better:

```suggestion
    geoCoordinates: [41.385532, 2.193823]
```

Reply:
Hah, yes the initial automated geolocation was maybe a bit rough 😅